client/src/components/Community/Communities/CommunityTemplate.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "./CommunityTemplate.css";
+
+function CommunityTemplate(props) {
```

Comment:
This component can be used as a generic template for individual community component.